### DISCUSSION TITLE
MEV - 礦工可提取價值

### DISCUSSION THREAD

**Discussion by yurenju** [lun-dao/LunDAO#76] [General]
這邊可以討論關於《[MEV - 礦工可提取價值][1]》文章的內容。

[1]: https://lundao.tech/blog/mev

**Comment by randylien** [comment]
用了非常淺顯易懂的文字解釋 MEV，太讚了！

**Reply by yurenju** [reply]
@randylien 好久不見，有機會多多交流啊 😎
